tools/bag_processing/readme.md

```diff
@@ -0,0 +1,36 @@
+\page bagprocessing Bag Processing
+
+# Package Overview
+The bag processing package provides several helper tools for handling bagfiles.
+
+# Usage Instructions
+For each script, run 'rosrun bag\_processing script\_name.py -h' for further details and 
+usage instructions.
+
+# Scripts
+## apply\_histogram\_equalization\_to\_images
```

Comment:
Single underscores have no special meaning in Markdown and do not need to be escaped like this. (Two or more underscores together is special, boldface or horizontal rule.)

However, as long as we're here, a nice style touch is to use backtick characters around `code_literals` and filenames so they render as monospace.

Reply:
Ok, I think when I didn't escape them it didn't render properly as an underscore? I think regardless the backtick solution is better so I'll update each of these with that

Reply:
Done